src/tui.rs

```diff
@@ -21,6 +22,7 @@ use tui::{
     Terminal,
 };
 
+pub fn render_tui(key: String, proj_name: &str) {
```

Comment:
Submodules are being passed as argument in main by being dynamically fetched from the remote repository but, you haven't taken that as argument